### PR TITLE
move all non-route content into lib/

### LINKED ISSUE #288: Move all non-route content into lib/
### Is your feature request related to a problem? Please describe.

It's awful to have to think about relative import paths(especially when refactoring code)

### Describe the solution you'd like

Make use of the lib alias/dynamic import path

### Describe alternatives you've considered

_No response_

### Additional context

https://rodneylab.com/sveltekit-lib/

### PR DESCRIPTION
**Pull request by nullxDEADBEEF**
# Description

Please include a summary of the change and which issue is fixed.

- Moved all non-route content into a lib/ folder
- Makes use of $lib import alias so we avoid nested import '../../x' in our code.

Fixes #288 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: frontend/src/lib/utils.ts [hunk 1]
```diff
@@ -36,7 +36,7 @@ export const mediaIdToUrlConverter = (
 }
 
 export const removeDuplicates = (arr: Media[]) => {
-  let unique_ids = []
+  const unique_ids = []
```
Comment (AndreasPB) [inline frontend/src/lib/utils.ts:39]: U crazy, CodeFactor

**Review by AndreasPB — APPROVED** [review]
Beautiful 🦄

File: frontend/src/lib/utils.ts (hunk 1 above)
Comment (AndreasPB) [reply frontend/src/lib/utils.ts:39]: I hope this still works as intended... Gotta admit I don't know the consequence of this is 🙈

Comment (nullxDEADBEEF) [reply frontend/src/lib/utils.ts:39]: Should we just yolo it? :D

Comment (AndreasPB) [reply frontend/src/lib/utils.ts:39]: Yea go for it :D

Comment (AndreasPB) [reply frontend/src/lib/utils.ts:39]: 🙏

Comment (nullxDEADBEEF) [reply frontend/src/lib/utils.ts:39]: Sweet 🙌

**Pull request merged**